file-availability.md

```diff
@@ -69,68 +70,143 @@ available.
 
 ``` {.sub}
-transfer_input_files = file1,../file2,/home/username/file3,dir1,dir2/
-```
+# my job submit file
+
+should_transfer_files = YES
+when_to_transfer_output = ON_EXIT
+transfer_input_files = file1, ../file2, /home/username/file3, dir1, dir2/
 
-Note: By default, the submit file \"executable\", \"output\", and
-\"error\" files are ALWAYS transferred.
+... other submit file details ...
 
-### Notes:
+```
 
--   DO NOT use \"transfer\_input\_files\" for files within /squid or
-    /staging as doing so will create severe performance issues for your
-    jobs and those of other users. (Similarly, you should never submit
-    jobs from within /squid or /staging.)
+By default, the submit file `executable`, `output`, and
+`error` files are **ALWAYS** transferred. 
+
+## Important Considerations
+
+-   **DO NOT use `transfer_input_files` for files within `/squid` or**
+    **`/staging` as doing so will create severe performance issues for your**
+    **jobs and those of other users. Jobs should should never be submitted**
+    **from within `/squid` or `/staging`.**
+    
+-   HTCondor\'s file transfer can cause issues for submit server performance
+    when too many jobs are transferring too much data at the same time.
+    Therefore, HTCondor file transfer is only good for input files up to
+    \~20 MB per file IF the number of concurrently-queued jobs will be 100
+    or greater. Even when individual files are small, there are issues when
+    the total amount of input data per-job approaches 500 MB. For cases
+    beyond these limitations, one of our other CHTC file delivery methods
+    should be used. Remember that creating a `tar.gz` file of directories
+    and files can give your input and output data a useful amount of
+    compression.
+    
 -   Comma-separated files and directories to-be-transferred should be
     listed with a path relative to the submit directory, or can be
-    listed with the absolute path(s), as shown above for \"file3\". The
-    submit file \"executable\" is automatically transferred and does not
-    need to be listed in \"transfer\_input\_files\".
+    listed with the absolute path(s), as shown above for `file3`. The
+    submit file `executable` is automatically transferred and does not
+    need to be listed in `transfer\_input\_files`.
+    
 -   All files that are transferred to a job will appear within the top
     of the working directory of the job, regardless of how they are
     arranged within directories on the submit server.
+    
 -   A whole directory and it\'s contents will be transferred when listed
-    without the \"/\" after the directory name. When a directory is
-    listed with the \"/\" after the directory name, only the directory
+    without the trailing forward slash (\"/\") after the directory name. When a directory is
+    listed with the trailing forward slash (\"/\") after the directory name, only the directory
```

Comment:
I feel like we should add a word of caution here about transferring a folder w/ 1000s of files but it doesn't need to be in this PR.

Reply:
should we instead recommend tarring up the dir?